### ISSUE TITLE
Closing drawer doesn't release body scroll block

### ISSUE THREAD

**Issue by ramononis** [posva/vue-mdc#227]
Clicking outside the drawer to close it doesn't remove the mdc-dialog-scroll-lock class from the body, making it impossible to scroll after using the drawer.

**Comment by ramononis** [comment]
[This test](https://github.com/posva/vue-mdc/blob/master/test/specs/Drawer.spec.js#L61) should also check if the scrollBlock is added/removed, which doesn't happen right now.

**Comment by posva** [comment]
Yes, it something that changed in the meantime but I still didn't update
